core/src/main/resources/application.conf

```diff
@@ -84,6 +84,10 @@ system {
 
   // Since the WorkflowLogCopyRouter is initialized in code, this is the number of workers
   number-of-workflow-log-copy-workers = 10
+
+  // Whether to run the metadata summarizer in this Cromwell instance.  There should only be one metadata summarizer
+  // per database, so if this Cromwell instance is being used in read-only mode this value should be `false`.
```

Comment:
I could see cases where people might want to have fine grained control, but I've also been advocating for giving fewer knobs to be turned. 

OTOH this isn't *truly* a 'read only' mode being discussed because they'll still want to submit workflows. It's more of a "don't actually run anything" mode and the metadata summarization is a side effect we need to take care of